### ISSUE TITLE
components in popover body don't receive events in tests

### ISSUE THREAD

**Issue by sandersoeteman** [sir-dunxalot/ember-tooltips#157]
This works fine in live usage, but only fails in tests. The following integration test fails with "Expected 5 assertions, but 4 were run". The dummy component is just a component.js with a class for the div.

```javascript
test('button in bare popover component receives events', function(assert) {
  assert.expect(5);

  this.on('click', function() {
    console.debug('button clicked!');
    assert.ok(true, 'the eventhandler is fired');
  });
  
  this.render(hbs`
    {{#dummy-component}}
      {{#popover-on-component}}
        <button class="my-button" {{action 'click'}}>test button</button>
      {{/popover-on-component}}
    {{/dummy-component}}
  `);

  const done = assert.async();

  let $target = $('.dummy-component');
  assertTooltipNotVisible(assert);
  triggerTooltipTargetEvent($target, 'mouseenter');
  assertTooltipRendered(assert);
  assertTooltipVisible(assert);
  
  let $button = $('.my-button');
  assert.equal($button.length, 1, 'the button can be found');
  $button.trigger('click');

  Ember.run.later(() => {
		done();
  }, 100);
});
```

I'm on ember(-cli) 2.10.0, ember-tooltips 2.7.1

**Comment by a15n** [comment]
is `console.debug('button clicked!');` being fired?

I think if you update this line it may work. 

```
let $target = $('.my-button');
```

Haven't tried reproducing. If it doesn't work I'll repro

**Comment by humanchimp** [comment]
Isn't this related to/a duplicate of #141? I believe that tests specify a `rootElement` other than the body, so the events go unhandled by Ember

**Comment by sandersoeteman** [comment]
@a15n changed the target selector and this didn't change the test result. The console message also isn't displayed. I got the dummy-component inspiration from your tests, have studied them to see if I did something wrong perhaps.
@humanchimp could be, as ember-tooltips functions fine in my live app.

**Comment by sir-dunxalot** [comment]
Fixed by #160

**Comment by sir-dunxalot** [comment]
Related to #141.

**Comment by sir-dunxalot** [comment]
Fixed by [2.9.0-beta.0](https://github.com/sir-dunxalot/ember-tooltips/releases/tag/2.9.0-beta.0).
